web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v3101/migrate-default.sql

```diff
@@ -1,2 +1,6 @@
+DELETE FROM cswservercapabilitiesinfo;
```

Comment:
Should be dropped the table and removed the related domain/persistence objects?

Reply:
Waiting for @jahow to see if that table is used somewhere or not.